### DISCUSSION TITLE
Why is the logo for Linkfree an animated rabbit ?

### DISCUSSION THREAD

**Discussion by Vedant-Manjrekar** [EddieHubCommunity/LinkFree#5175] [Q&A] [answered]
I would like to know why is the logo for "Linkfree" a rabbit, I believe the website is about displaying relevant links of users and hence a logo related to links or something similar would make more sense.

As a constructive feedback I would say (my personal opinion) that the logo feels kind of immature and makes one doubt the credibility of the website despite the site being exceptionally excellent in terms of UI/UX and overall feel and features.

Lastly, this is just my personal opinion and feedback and I have no intension of demeaning the creators and contributors of this excellent open-source project.

**Comment by schmelto** [comment, marked as answer]
We currently have the rabbit as logo due its the mascot of the eddiehub community. But we are looking for a logo in this discussion https://github.com/EddieHubCommunity/LinkFree/discussions/206
